Hearthstone Deck Tracker/Controls/DeckImportingControl.xaml.cs

```diff
@@ -112,6 +112,31 @@ private void CheckBoxImportAll_OnModified(object sender, RoutedEventArgs e)
 			}
 		}
 
+		private void CheckBoxImport_OnChecked(object sender, RoutedEventArgs e)
+		{
+			int modifiedCount = 0;
+
+			if (ItemsControl == null)
+				return;
+
+			foreach(var item in ItemsControl.Items)
+			{
+				var c = (ContentPresenter)ItemsControl.ItemContainerGenerator.ContainerFromItem(item);
+				var chk = (CheckBox)c.ContentTemplate.FindName("CheckBoxImport", c);
+                if (chk.IsChecked.Value == true)

```

Comment:
` == true` is redundant when looking at a boolean.
This should be either `chk.IsChecked.HasValue && chk.IsChecked.Value`, `chk.IsChecked ?? false` or `chk.IsChecked == true` (in this case it's a `bool?`).